server/src/format.ts

```diff
@@ -10,11 +10,14 @@ export default async function format(
     return await exec(execPath, ['format', '-s', identWidth.toString()], text)
   } catch (errors) {
     const errorMessage =
-      "prisma-fmt error'd during formatting. This was likely due to a syntax error. Please see linter output."
+      "prisma-fmt error'd during formatting. This was likely due to a syntax error. Please see Prisma Language Server output."
```

Comment:
Not sure about others, but I usually don't have that 2nd pane open.

Can we make this more like instructions? Something like: 

To see the output, go to View > Output from the toolbar, then select "Prisma Language Server"

Reply:
Is there anyway to show the error inside the toast notification or is that too verbose?

Reply:
Yeah probably most people don't have that open. While it is possible to show the error inside the toast notification, I think that would be too verbose.

But I agree, instructions would be very helpful.